### PR TITLE
Init model

### PR DESCRIPTION
**Pull request by HUSTToanLH**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
-
-group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
+source "https://rubygems.org"
+
+gem "rails", "4.2.5"
```
Comment (dieunb) [inline Gemfile]: `4.2.6`

File: Gemfile [hunk 2]
```diff
@@ -1,47 +1,42 @@
...
+gem "bcrypt", "3.1.7"
+gem "faker", "1.4.2"
+gem "carrierwave", "0.10.0"
+gem "mini_magick", "3.8.0"
+gem "fog", "1.36.0"
+gem "will_paginate", "3.0.7"
```
Comment (dieunb) [inline Gemfile]: Dùng gem kaminari in this project

File: Gemfile [hunk 3]
```diff
@@ -1,47 +1,42 @@
...
+gem "bootstrap-will_paginate", "0.0.10"
```
Comment (dieunb) [inline Gemfile]: remove it

**Comment by hitshootnum1** [comment]
ok

**Comment by QuangVinh93** [comment]
ok

**Pull request merged**